dev-support/benchmark/grouping/grouping.sh

```diff
@@ -81,12 +86,12 @@ do
 		    echo "OUTPUT: ${OUT}"
 		    echo "PARALLELISM: ${P}"
  		    echo "========="
-		    /usr/local/hadoop-2.5.2/bin/hdfs dfs -rm -r ${OUT}
+		    ${HDFS} dfs -rm -r ${OUT}
 		    INPUT="hdfs://${I}"
 		    OUTPUT="hdfs://${OUT}"
 		    AGGS="-vagg ${VAGG} ${VAK} ${VARK} -eagg ${EAGG} ${EAK} ${EARK}"
 		    ARGS="-csv ${CSV} -vgk ${VGK} -egk ${EGK} ${AGGS}"
```

Comment:
`-vgk` and `-egk` must only be present when VGK and/or EGK are given (non-empty)